src/main/java/com/starbucks/model/Order.java

```diff
@@ -1,81 +1,50 @@
 package com.starbucks.model;
 
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import java.util.UUID;
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
 
+import javax.persistence.*;
+import java.util.UUID;
 
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter
+@Getter
+@Builder
+@Table(name = "Order", schema = "starbucks", catalog = "postgres")
 public class Order {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq")
+    @Column(name = "id", nullable = false)
```

Comment:
Id is nullable=false by default there is no need in pointing it here, it's redundant